vncdotool/client.py

```diff
@@ -131,7 +133,9 @@ class VNCDoToolClient(rfb.RFBClient):
     cmask = None
 
     def _decodeKey(self, key):
-        if self.factory.force_caps and key.isupper():
+        if (self.factory.force_caps and key.isalpha() and key.islower()) \
```

Comment:
ok, I like points 3 and 4, with the aim of fixing special keys #26 but I don't understand why the CAPS LOCK case inversion behaviour is desirable.  --force-caps is intended to work around vncservers that aren't properly handling upper case letters.